src/main/java/com/softserve/demo/util/ProviderMapper.java

```diff
@@ -10,32 +10,19 @@
 @Mapper(componentModel = "spring")
 public interface ProviderMapper {
     @Mappings({
-            @Mapping(target = "id", source = "id"),
-            @Mapping(target = "name", source = "name"),
-            @Mapping(target = "email", source = "email"),
-            @Mapping(target = "description", source = "description"),
-            @Mapping(target = "image", source = "image"),
-            @Mapping(target = "lastUpdate", source = "lastUpdate"),
             @Mapping(target = "userDTO", source = "user"),
 //            @Mapping(target = "ordersDTO", source = "orders"),
 //            @Mapping(target = "servicesDTO", source = "services"),
             @Mapping(target = "location", source = "location")
             //TODO: create mappping for orders, services, locationsgit status
```

Comment:
I know that it is not yours. But please can you delete this git status and delete 1 letter 'p' from mapping. My eyes are bleeding!

Reply:
Sure